vendor/github.com/knative/pkg/test/logging/logging.go

```diff
@@ -58,9 +59,9 @@ type BaseLogger struct {
 // ExportView will emit the view data vd (i.e. the stats that have been
 // recorded) to the zap logger.
 func (e *zapMetricExporter) ExportView(vd *view.Data) {
-	// We are not curretnly consuming these metrics, so for now we'll juse
+	// We are not currently consuming these metrics, so for now we'll juse
 	// dump the view.Data object as is.
-	e.logger.Info(vd)
+	e.logger.Debug(spew.Sdump(vd))
```

Comment:
This is a change in the vendor directory. It'll need to be updated in knative/pkg and then pulled in here.

Reply:
argh, missed that :|